Formula/pnpm.rb

```diff
@@ -20,11 +20,65 @@ class Pnpm < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "efcac832dc165a64bbb93b7018876cf7dc70d669ae3b1f2a68c46ea0bcddc572"
   end
 
-  depends_on "node"
+  depends_on "node" => :build
+
+  PNPM_BUILDTIME_VERSION = "v6.7".freeze
+
+  resource "pnpm-buildtime" do
+    url "https://get.pnpm.io/#{PNPM_BUILDTIME_VERSION}.js"
```

Comment:
```suggestion
  resource "pnpm-buildtime" do
    url "https://get.pnpm.io/v6.7.js"
```
We don't need the constant here.

Reply:
Thanks!
This file is placed in https://github.com/pnpm/get so that I will add this URL as a comment.